<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseLogRegScreen</class>
 <widget class="QDialog" name="chooseLogRegScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Вход и регистрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="loginScreen" name="loginPage"/>
     <widget class="registerScreen" name="regPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>loginScreen</class>
   <extends>QWidget</extends>
   <header>loginScreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>registerScreen</class>
   <extends>QWidget</extends>
   <header>registerScreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
